Makefile

```
@@ -13,6 +13,15 @@ ifndef UNAME_M
 UNAME_M := $(shell uname -m)
 endif
 
+ARCH_LINUX1 := $(shell grep "Arch Linux" /etc/os-release 2>/dev/null)
+ARCH_LINUX2 := $(shell grep "ID_LIKE=arch" /etc/os-release 2>/dev/null)
+ifdef ARCH_LINUX1
+ARCH_ADD = -lcblas
+endif
+ifdef ARCH_LINUX2
+ARCH_ADD = -lcblas
+endif
```

grep is quite smart and we can leverage this:

```suggestion
ARCH_LINUX := $(shell grep -q -e "Arch Linux" -e "ID_LIKE=arch" /etc/os-release && echo 1)
ifdef ARCH_LINUX
LDFLAGS_EXTRA += -lcblas
endif
```

---

Makes sense. I didn't know that you could grep multiple strings.